packages/docs/src/components/header/components/algolia-search/AlgoliaSearch.vue

```diff
@@ -27,30 +9,22 @@ import { Options, Vue } from 'vue-class-component'
   name: 'Search',
 })
 export default class Search extends Vue {
```

Comment:
* Use composition api/class component syntax.
* File name and component name should be consistent.

Reply:
Fixed